src/caret_analyze/infra/lttng/lttng_info.py

```diff
@@ -855,8 +855,6 @@ def _get_timers(self, node_id: str) -> list[TimerValue]:
         for _, row in tim.df.iterrows():
             if row['node_id'] != node_id:
                 continue
-            if row['callback_id'] is pd.NA:
-                continue
```

Comment:
@xygyo77 
For what reason is this process being removed?
It appears that the process is being deleted for reasons unrelated to the content of this PR amendment.